events/suite_test.go

```diff
@@ -46,13 +48,18 @@ func Test(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
+	ctx = context.Background()
+	ctx = log.IntoContext(context.Background(), ginkgo.GinkgoLogr)
```

Comment:
Did you mean to create a new background context here?

Reply:
Cleaning that up with a few updates to the tests